ISeriesProvider/DB2iSeriesSchemaProvider.cs

```diff
@@ -307,27 +339,54 @@ public static void SetColumnParameters(ColumnInfo ci, long? size, int? scale)
 			}
 		}
 
-        #endregion
-
-	    private string GetLibList(DataConnection dataConnection)

```

Comment:
Refactored LibList to include connection string fallback for iDB2 and connection string for DB2